doc/source/conf.py

```diff
@@ -64,6 +64,9 @@
 numpydoc_show_class_members = False
 nbsphinx_timeout = 120
 
+# Autodoc settings
+autodoc_typehints = "description"
```

Comment:
This worked as expected on my local html build. 

However, I had to upgrade the package version of Sphinx to see the effect of this relatively new option (I was using the Ubuntu's ancient package python3-sphinx version 1.8.5-7ubuntu3). It now works with Sphinx upgraded to version 4.1.2. Not sure if this has be to considered in `requirements/docs.pip`.

Reply:
I've forced RTD to use a sphinx version > 4.0. 
https://scikit-rf.readthedocs.io/en/latest/

What about now ?